src/main/java/Storage.java

```diff
@@ -0,0 +1,110 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+
+    protected final File TASKFILE;
+    protected String createResult = "";
+
+    // constructor
+    public Storage(String filePath) {
+
+        this.TASKFILE = new File(filePath);
+
+        try {
+            if (TASKFILE.createNewFile()) {
+                // if a tasks.txt file does not exist, we create a file so that we can read from it in the future
+                this.createResult = "     Duke has noticed that you do not have a text file to store your tasks!\n" +
```

Comment:
Seems like an unnecessary class member. Maybe declare a local variable if you do want to use it since it is not used elsewhere in the class?